drivers/wifi/esp32/src/esp_wifi_drv.c

```diff
@@ -619,14 +619,21 @@ static int esp32_wifi_status(const struct device *dev, struct wifi_iface_status
 			status->rssi = ap_info.rssi;
 			memcpy(status->bssid, ap_info.bssid, WIFI_MAC_ADDR_LEN);
 
-			if (ap_info.phy_11b) {
-				status->link_mode = WIFI_1;
-			} else if (ap_info.phy_11g) {
-				status->link_mode = WIFI_3;
-			} else if (ap_info.phy_11n) {
-				status->link_mode = WIFI_4;
-			} else if (ap_info.phy_11ax) {
-				status->link_mode = WIFI_6;
+			wifi_phy_mode_t phy_mode;
+			esp_err_t err;
```

Comment:
Please introduce variables at the start of the block

Reply:
Got it.